.circleci/cimodel/data/pytorch_build_data.py

```diff
@@ -100,6 +104,7 @@ def child_constructor(self):
 
         next_nodes = {
             "xenial": XenialCompilerConfigNode,
+            "bionic": BionicCompilerConfigNode,
```

Comment:
@ezyang Hmmm so I could make `xenial` and `bionic` use the same CompilerConfigNode here, but I'm not sure whether we want to keep it separate for different distros.

Reply:
idk, maybe @kostmo would know. cc @seemethere too

Reply:
If the behavior forks at some point between distros, will need separate nodes, but either way is fine.